common/common.cpp

```diff
@@ -1386,3 +1387,66 @@ void dump_non_result_info_yaml(FILE * stream, const gpt_params & params, const l
     fprintf(stream, "typical_p: %f # default: 1.0\n", sparams.typical_p);
     fprintf(stream, "verbose_prompt: %s # default: false\n", params.verbose_prompt ? "true" : "false");
 }
+
+//
+// KV cache utils
+//
+
+void dump_kv_cache_view(const llama_kv_cache_view & view, int row_size) {
+    printf("=== Dumping KV cache. total cells %d, max sequences per cell %d, populated cells %d, total tokens in cache %d\n",
```

Comment:
Isn't it better to use `LOG_TEE` instead of `printf`?

Reply:
Sure, I can make that change. I find the logging behavior sort of confusing. I.E. if I compile without logging I'll see the `LOG_TEE` output. However, if I compile with logging but `--disable-logs` it will just never show up on the console.

Reply:
Hmm, actually... Using the LOG stuff is possibly sort of weird if I'm printing stuff out in pieces rather than line by line like those functions do. What do you think, still change it?